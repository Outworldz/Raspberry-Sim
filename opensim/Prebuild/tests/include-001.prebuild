<?xml version="1.0" encoding="utf-8" ?> 
<Prebuild xmlns="http://dnpb.sourceforge.net/schemas/prebuild-1.7.xsd" version="1.7">
  <Solution name="include-001" version="1.2.3">
    <Configuration name="Debug">
      <Options>
	<CompilerDefines>DEBUG;TRACE</CompilerDefines>
	<OptimizeCode>false</OptimizeCode>
	<OutputPath>bin/Debug</OutputPath>
	<DebugInformation>true</DebugInformation>
      </Options>
    </Configuration>
    <Configuration name="Release">
      <Options>
	<CompilerDefines>TRACE</CompilerDefines>
	<OutputPath>bin/Release</OutputPath>
	<OptimizeCode>true</OptimizeCode>
	<DebugInformation>false</DebugInformation>
      </Options>
    </Configuration>
    <?include file="include-001.include"?>
  </Solution>
</Prebuild>
