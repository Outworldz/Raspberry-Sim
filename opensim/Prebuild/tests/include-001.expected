<Prebuild xmlns="http://dnpb.sourceforge.net/schemas/prebuild-1.7.xsd" version="1.7">
  <Solution name="include-001" version="1.2.3">
    <Configuration name="Debug">
      <Options>
        <CompilerDefines>DEBUG;TRACE</CompilerDefines>
        <OptimizeCode>false</OptimizeCode>
        <OutputPath>bin/Debug</OutputPath>
        <DebugInformation>true</DebugInformation>
      </Options>
    </Configuration>
    <Configuration name="Release">
      <Options>
        <CompilerDefines>TRACE</CompilerDefines>
        <OutputPath>bin/Release</OutputPath>
        <OptimizeCode>true</OptimizeCode>
        <DebugInformation>false</DebugInformation>
      </Options>
    </Configuration>
    <Project name="Prebuild" path="src" language="C#" assemblyName="include-001" type="Exe" version="1.2.3" xmlns="http://dnpb.sourceforge.net/schemas/prebuild-1.7.xsd">
      <Configuration name="Debug">
        <Options>
          <CompilerDefines>DEBUG;TRACE</CompilerDefines>
          <OptimizeCode>false</OptimizeCode>
          <OutputPath>bin/Debug</OutputPath>
          <DebugInformation>true</DebugInformation>
          <KeyFile>Prebuild.snk</KeyFile>
          <SuppressWarnings>1595</SuppressWarnings>
        </Options>
      </Configuration>
      <Configuration name="Release">
        <Options>
          <CompilerDefines>TRACE</CompilerDefines>
          <OutputPath>bin/Release</OutputPath>
          <OptimizeCode>true</OptimizeCode>
          <DebugInformation>false</DebugInformation>
          <KeyFile>Prebuild.snk</KeyFile>
          <SuppressWarnings>1595</SuppressWarnings>
        </Options>
      </Configuration>
      <Reference name="System" />
      <Files>
        <Match pattern="*.cs" recurse="true" />
      </Files>
    </Project>
  </Solution>
</Prebuild>
